docs/2.6.0/README.md

```diff
@@ -0,0 +1,123 @@
+# Working on the documentation
+
+This folder manages the full build of technical documentation from the following repos:
+
+* [Daml docs](https://github.com/digital-asset/daml/tree/main/docs)
+* [Canton docs](https://github.com/DACH-NY/canton/tree/main/docs)
+* [Daml-Finance docs](https://github.com/digital-asset/daml-finance/tree/main/docs)
+
+:bulb: Docs in this repo build the main TOC with pointers to pages from the above repos.
+
+The process for updating the docs TOCs has some potential pitfalls. Follow the instructions in this README *carefully* and in order.
+
+- [Setting up this repo locally](https://github.com/digital-asset/docs.daml.com#setting-up-this-repo-locally)
+- [Build and view the docs locally](https://github.com/digital-asset/docs.daml.com#build-and-view-the-docs-locally)
+- [How to commit changes to the docs](https://github.com/digital-asset/docs.daml.com#how-to-update-the-docs)
+
+:arrow_right: If you encounter any issues, reach out to #product-docs on Slack.
+
+### Live preview
+
+Run the `live-preview` script to render a local view of the site.
+
+```zsh
+live-preview
+```
+
+The html files are in `workdir/build/gen`. Rerun the script to view any edits to `docs/` files or the top-level ToC (`_toc.yml`).
+
+:warning: If you make changes to docs managed by the other repos, you have to commit them there. In this repository, you can only make changes to the top-level ToC (`_toc.yml`) or some section introduction pages in the `docs` folder.
+
+:warning: If you do make a change to the TOCs in this repo, you then have to duplicate the change in the relevant satellite repo (except for the daml repo, which does not have a local ToC).
+
+### Full build
+
+To build the PDF docs or the exact HTML files that we publish to [docs.daml.com](https://docs.daml.com), run the full build locally:
+
+```zsh
+build
+```
+
+The HTML files will be in `workdir/build/sphinx-target/html`; you can view them
+by doing something like:
+
+```zsh
+$ cd workdir/build/sphinx-target/html
+$ python -m http.server 8000 --bind 127.0.0.1
+```
+
+Then pointing your web browser to [http://localhost:8000](http://localhost:8000).
```

Comment:
If this section gets deleted, we still need this line after running live-preview:

Then point your web browser to [http://localhost:8000](http://localhost:8000).